2024/02-data-comm/slides.qmd

```diff
@@ -15,6 +15,7 @@ css: custom.css
 -   Implement principles of effective design using {ggplot}. Including:
 -   Maximize data:ink ratio using themes and facets
```

Comment:
Minor formatting thing, but I think some of these bullets are supposed to indented one level?